nixos/modules/tasks/filesystems/bcachefs.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  inInitrd = any (fs: fs == "bcachefs") config.boot.initrd.supportedFilesystems;
+
+in
+
+{
+  config = mkIf (any (fs: fs == "bcachefs") config.boot.supportedFilesystems) {
+
+    system.fsPackages = [ pkgs.bcachefs-tools ];
```

Comment:
If somebody builds an installer image using:

```
{
  boot.supportedFilesystems = [ "bachefs" ];
}
```

would not the expectation be that, this installer actually supports bcachefs?